README.md

```diff
@@ -102,12 +112,23 @@ spec:
 | `LEGO_KUBE_API_URL` | n | `http://127.0.0.1:8080` | API server URL |
 | `LEGO_LOG_LEVEL` | n | `info` | Set log level (`debug|info|warn|error`) |
 
-
 ## Full deployment examples
 
 - [Nginx Ingress Controller](examples/nginx/)
 - [GCE Load Balancers](examples/gce/)
 
+## Troubleshooting
+
+When interacting with *kube-lego*, its a good idea to run with `LEGO_LOG_LEVEL=debug` for more verbose details.
+Additionally, be aware of the automatically created resources (see environment variables) when cleaning up or testing.
+
+Possible resources for *help*:
+
+* `#coreos` on freenode
```

Comment:
I think that there's a big difference between "help" and "support". While the wording of this could be slightly adjusted to let users know not the **expect** support, I don't think there's any problem with informing users where they might be able to get a helping hand, or at least some advice. From what I've seen, kube-lego seems to be used a fair amount in the Kubernetes community, so I don't think there's going to be much of an issue directing people to a general Kube room.